charger/warp/types.go

```diff
@@ -97,11 +97,11 @@ const (
 	ExternalControlCurrentlySwitching
 )
 
-type EmState struct {
-	ExternalControl ExternalControl `json:"external_control"`
-	PhasesSwitched  int             `json:"phases_switched"`
-	Input3State     bool            `json:"input3_state"`
-	Input4State     bool            `json:"input4_state"`
-	RelayState      bool            `json:"relay_state"`
-	ErrorFlags      int             `json:"error_flags"`
+type PMState struct {
```

Comment:
Muss das hier geändert werden?

Reply:
> Muss das hier geändert werden?

In dem letzten großen Refactoring von Tinkerforge wurden die Topics in "energy_manager" und "power_manager" getrennt. Für die Phasenumschaltung sind nur noch die beiden in "power_manager" relevant.

Reply:
Das Topic hier ist ja schon power_manager, also alles richtig bis auf den Namen. Falls Du den ändern willst bitte separater PR, dann aber auch konsequent an allen Stellen ;)

Reply:
Alles klar, dann lasse ich das hier erstmal so bestehen und füge nur den low_level_state hinzu ;-)